sig/datadog/di/probe_notification_builder.rbs

```diff
@@ -16,9 +16,9 @@ module Datadog
 
       def build_errored: (Probe probe, Exception exception) -> Hash[Symbol,untyped]
 
-      def build_executed: (Probe probe, ?trace_point: TracePoint, ?rv: untyped?, ?duration: Float, ?caller_locations: Array[untyped], ?args: untyped?, ?kwargs: untyped?, ?serialized_entry_args: untyped?) -> Hash[Symbol,untyped]
+      def build_executed: (Probe probe, ?instance_vars: untyped?, ?rv: untyped?, ?duration: Float, ?caller_locations: Array[untyped], ?args: untyped?, ?kwargs: untyped?, ?serialized_entry_args: untyped?) -> Hash[Symbol,untyped]
```

Comment:
You have a definition by instrumenter as `Hash[untyped, untyped]`. I would suggest to use type alias defined in Instrumenter class